identity-verification/module_2/2.3 Authentication IDV API Example.ipynb

```diff
@@ -347,7 +347,7 @@
     "<div class=\"alert alert-info\"><strong> Results </strong>\n",
     "    \n",
     "\n",
-    "- Check out the similarity of each face found, it should range from 99.99 to 99.96 (Rekognition is extreemly accurate) \n",
+    "- Check out the similarity of each face found, it should range from 99.99 to 99.96 (Rekognition is extremly accurate) \n",
```

Comment:
```suggestion
    "- Check out the similarity of each face found, it should range from 99.99 to 99.96 (Rekognition is extremely accurate) \n",
```

I've addressed a typo, but I'd consider remove the sentence _Rekognition is extremely accurate_. It's a bit fluffy

Reply:
+1 lets remove it.